### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by achingbrain** [testground/sdk-js#20]
The read me says:

> Usage
> See testground.github.io/sdk-js.

..but https://testground.github.io/sdk-js 404s and there's no gh-pages branch in this repo.

**Comment by hacdias** [comment]
Indeed, I wrote that there because I was planning to release a v0.x at the time. I just released v0.1.0 and it seems AEGir release with docs is broken: https://github.com/ipfs/aegir/issues/881

I will put them up as soon as we can release them!
